src/Console/FieldGeneratorCommand.php

```diff
@@ -22,4 +26,42 @@ protected function getOptions(): array
             ['full', 'F', InputOption::VALUE_NONE, 'Include the seldom needed resolver arguments $context and $resolveInfo'],
         ];
     }
+
+    protected function handleTestCreation($path)
+    {
+        if (! $this->option('test') && ! $this->option('pest')) {
```

Comment:
This only works with `'test'`, we should either implement `pest` support or warn that it does not work.